From f5c04fae49ef13ee4a6c59c4246657ac632fa67b Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Thu, 26 Nov 2020 18:47:37 +0000
Subject: [PATCH 09/25] Allow cross compiling for Darwin

---
 configure    | 3 +++
 configure.ac | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/configure b/configure
index 2cb22a7806f..04b8bc17525 100755
--- a/configure
+++ b/configure
@@ -4112,6 +4112,9 @@ then
 	*-*-linux*)
 		ac_sys_system=Linux
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		;;
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
diff --git a/configure.ac b/configure.ac
index 967d34d47a3..9b6b3a92c33 100644
--- a/configure.ac
+++ b/configure.ac
@@ -324,6 +324,9 @@ then
 	*-*-linux*)
 		ac_sys_system=Linux
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		;;
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
-- 
2.51.0

