From 6ad8365199c579090391bcb4fce65ae62b18b3e5 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Thu, 26 Nov 2020 18:47:37 +0000
Subject: [PATCH 09/24] Allow cross compiling for Darwin

---
 configure    | 3 +++
 configure.ac | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/configure b/configure
index 345822a6c6d..60bc25db247 100755
--- a/configure
+++ b/configure
@@ -4110,6 +4110,9 @@ then
 	*-*-linux*)
 		ac_sys_system=Linux
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		;;
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
diff --git a/configure.ac b/configure.ac
index d6059471771..8b71e1450c2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -324,6 +324,9 @@ then
 	*-*-linux*)
 		ac_sys_system=Linux
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		;;
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
-- 
2.45.2

