From efecf299dcba2943b9955988197f0e8f8c863143 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Thu, 26 Nov 2020 18:47:37 +0000
Subject: [PATCH 09/21] Allow cross compiling for Darwin

---
 configure    | 3 +++
 configure.ac | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/configure b/configure
index de426e6b68..58b46ecb22 100755
--- a/configure
+++ b/configure
@@ -4041,6 +4041,9 @@ then
 	*-*-linux*)
 		ac_sys_system=Linux
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		;;
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
diff --git a/configure.ac b/configure.ac
index 8eb9676748..c0321397df 100644
--- a/configure.ac
+++ b/configure.ac
@@ -324,6 +324,9 @@ then
 	*-*-linux*)
 		ac_sys_system=Linux
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		;;
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
-- 
2.39.3 (Apple Git-146)

