From 6c2667457a3910e23072d30f7c2f25336e38117e Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Tue, 5 Sep 2023 21:51:31 +0200
Subject: [PATCH 20/22] Override configure LIBFFI

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index e24c3babcfe..24da75c7bf3 100755
--- a/configure
+++ b/configure
@@ -15252,7 +15252,7 @@ if test "x$ac_cv_lib_ffi_ffi_call" = xyes
 then :
 
                 have_libffi=yes
-        LIBFFI_CFLAGS="-I${SDKROOT}/usr/include/ffi -DUSING_APPLE_OS_LIBFFI=1"
+        LIBFFI_CFLAGS="-I${PREFIX}/include -DUSING_APPLE_OS_LIBFFI=0"
         LIBFFI_LIBS="-lffi"
 
 fi
-- 
2.39.5 (Apple Git-154)

