From 9a026370bd659c0214389f45b41d6eeffd4c368e Mon Sep 17 00:00:00 2001
From: Eric Lundby <Eric.Lundby@gmail.com>
Date: Mon, 9 Mar 2026 15:28:24 -0600
Subject: [PATCH] Allow setting llvm version

---
 Tools/jit/_llvm.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Tools/jit/_llvm.py b/Tools/jit/_llvm.py
index f09a8404871..726c7f1dbfd 100644
--- a/Tools/jit/_llvm.py
+++ b/Tools/jit/_llvm.py
@@ -10,7 +10,7 @@
 
 import _targets
 
-_LLVM_VERSION = 19
+_LLVM_VERSION = int(os.environ.get("LLVM_VERSION", "19"))
 _LLVM_VERSION_PATTERN = re.compile(rf"version\s+{_LLVM_VERSION}\.\d+\.\d+\S*\s+")
 _EXTERNALS_LLVM_TAG = "llvm-19.1.7.0"
 
-- 
2.53.0.windows.1

